docker-compose.yml

```diff
@@ -41,7 +41,7 @@ services:
       - "28017:28017"
 
   mysql-8:
-    image: mysql:8
+    image: mysql:8-oracle
```

Comment:
For anyone who uses this branch, you'll need to make sure you use Docker's debug feature to purge all the images before you attempt using MySQL. Without doing so, you will have odd behaviour in regards to Whitehall connectivity and replicating data.